universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl

```diff
@@ -397,7 +397,7 @@
     </choose>
   </macro>
   <macro name="locator">
-    <group delimiter=" " prefix=", " suffix=".">
+    <group delimiter="&amp;#x00A0;" prefix=", " suffix=".">
```

Comment:
That's not right, you're escaping the ampersand. Also, use the decimal escaping for readability
```suggestion
    <group delimiter="&#160;" prefix=", " suffix=".">
```

Reply:
Done.